src/components/FormatStatsRender.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC } from "react";
+import { useMemo } from "react";
+import { RepositoryContributionsCard } from "./RepositoryContributionsCard";
+import {
+  PullRequestContributionsByRepository,
+  RepositoryRenderFormat,
+} from "@/types/github";
+import ExportDropdown from "./ExportDropdown";
+import { generateText } from "@/utils/generateText";
+import { exportAsJSON, exportAsText } from "@/utils/exportRepositories";
+import { FaRegFolderOpen } from "react-icons/fa";
+
+interface NoContributionsProps {
+  message: string;
+}
+
+const NoContributions: FC<NoContributionsProps> = ({ message }) => (
+  <div className="flex flex-col items-center justify-center">
+    <FaRegFolderOpen size={28} />
+    <p className="text-xl">{message}</p>
+  </div>
+);
+
+interface FormatStatsRenderProps {
+  repositories: PullRequestContributionsByRepository[];
+  format: RepositoryRenderFormat;
+  exportJSON: () => void;
+  exportText: () => void;
+}
+
+const FormatStatsRender: FC<FormatStatsRenderProps> = ({
+  repositories,
+  format,
+}) => {
```

Comment:
`FormatStatsRenderProps` has exportJSON and exportText but you're not using them here.